libs/shared/src/consts/providers/provider.enum.ts

```diff
@@ -49,6 +49,7 @@ export enum SmsProviderIdEnum {
 export enum ChatProviderIdEnum {
   Slack = 'slack',
   Discord = 'discord',
+  MsTeams = 'msteams',
```

Comment:
Not really important right now, but I feel it should be MS_TEAMS: '..' per our enum conventions.